### ISSUE TITLE
Error when using ignored values in destructuring assignments

### ISSUE THREAD

**Issue by dermuecke** [gajus/eslint-plugin-jsdoc#716]
There is an error when using ignored values in destructuring assignments.

[Ignoring some returned values (MDN)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#ignoring_some_returned_values)

## Expected behavior
No error expected

## Actual behavior
```
TypeError: Cannot read property 'type' of null
[...] 
in eslint-plugin-jsdoc/dist/jsdocUtils.js:217:29
```

## Example
```js
/**
 * @param b
 */
const foo = ([, b]) => b;
```

## Environment

- Node version: 14.15.5
- ESLint version: 6.8.0
- `eslint-plugin-jsdoc` version: 32.3.0

**Comment by brettz9** [comment]
Thanks for the report!
